source/winAPI/sessionTracking.py

```diff
@@ -445,6 +271,8 @@ def _getSessionLockedValue() -> WTS_LockState:
 	with WTSCurrentSessionInfoEx() as info:
 		infoEx: WTSINFOEX_LEVEL1_W = info.contents.Data.WTSInfoExLevel1
 		sessionFlags: ctypes.wintypes.LONG = infoEx.SessionFlags
-		lockState = WTS_LockState(sessionFlags)
-		log.debug(f"Query Lock state result: {lockState!r}")
+		try:
+			lockState = WTS_LockState(sessionFlags)
+		except ValueError:

```

Comment:
A log message or comment about why this might fail? Do we know? If not, it might be helpful to refer to the issue where it was reported.

Reply:
This is because in some cases `sessionFlags=-0x1` is being returned, causing #14379. `-0x1` does not match any value in the enum.
Also, the winAPI notes that the lock state can be one or more of these values. This means that the `IntEnum` should really be an `IntFlag`, and that we could get a value `WTS_SESSIONSTATE_UNKNOWN | WTS_SESSIONSTATE_LOCK | WTS_SESSIONSTATE_UNLOCK`. However, I think we should continue to handle any mixed state as an error still, rather than implementing the `IntFlag`.
I'm going to add a comment explaining this.

https://learn.microsoft.com/en-us/windows/win32/api/wtsapi32/ns-wtsapi32-wtsinfoex_level1_w